apiserver/params/actions.go

```diff
@@ -51,7 +51,7 @@ type Action struct {
 // EnqueuedActions represents the result of enqueuing actions to run.
 type EnqueuedActions struct {
 	OperationTag string         `json:"operation"`
-	Actions      []StringResult `json:"actions,omitempty"`
+	Actions      []ActionResult `json:"actions,omitempty"`
```

Comment:
Shouldn't we still bump the facade version, even if we end up removing older versions for a 3.0 release?